### DISCUSSION TITLE
New release to fix breaking bugs

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2406] [General]
Since we have https://github.com/OpenMage/magento-lts/issues/2399 and https://github.com/OpenMage/magento-lts/pull/2364 I think we should release a new version, the version bump is already merged so I guess we only have to merge https://github.com/OpenMage/magento-lts/pull/2370 and publish.

**Comment by sreichel** [comment]
Agree.

**Reply by fballiano** [reply]
@sreichel https://github.com/OpenMage/magento-lts/pull/2370 is waiting for reviews ;-)
sorry didn't mean to be pushy :-)

**Comment by addison74** [comment]
Maybe in such situations when quick changes are needed to fix serious issues a patch file would be more useful. A new version can be created too but I think that the effort made is greater. If I remember well the Magento team in the past used patch files.

**Reply by sreichel** [reply]
I know we do not follow semantic versioning, but this would be a hotfix-release ...

Not the first time we bumped Version to sort out a newly introduced bug.

**Reply by fballiano** [reply]
@ADDISON74 a patch could be a nice idea but I don't think we've a way to "publish" it here on github or anywhere (on the website? who will do it? we have no write privileges on that)

@sreichel absolutely, that's why I tried to fix immediately, get the PRs ready and did a version bump only for the v20, but all of that is still waiting for reviews... actually I closed the first version bump PR since now it's already too late and a late hotfix didn't make sense anymore IMHO.
For these time sensitive thing only a real maintainer can make the difference, we normal contributors can't do much since we can't really merge much into the official branches.
